zellij-server/src/screen.rs

```diff
@@ -1654,7 +1658,7 @@ pub(crate) fn screen_thread_main(
                 active_tab_and_connected_client_id!(
                     screen,
                     client_id,
-                    |tab: &mut Tab, client_id: ClientId| tab.move_focus_left(client_id),
+                    |tab: &mut Tab, client_id: ClientId| tab.move_focus_left(client_id, true), // set wrap around flag to true does not cause problem since the action does not change tab
```

Comment:
Let's make it `false` then... we can't know what this will be (mis)used for in the future :)